charts/osm/crds/meshconfig.yaml

```diff
@@ -180,3 +180,16 @@ spec:
                       description: Sets the service certificate validity duration, represented as a sequence of decimal numbers each with optional fraction and a unit suffix.
                       type: string
                       default: "24h"
+                featureFlags:
+                  description: OSM feature flags
+                  type: object
+                  properties:
+                    enableWASMStats:
+                      type: boolean
+                      default: true
```

Comment:
Should this be false by default? In the struct definition below, `EnableWASMStats` is a bool, which makes it false by default. Hope we can make it consistent.

Reply:
In `values.schema.json` the default values for `enableWASMStats` and `enableEgressPolicy` are true, so that's why I decided to set the default values to true here. I understand what you are saying though about the inconsistency with the struct below.